.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "4"
```

Comment:
Please note it won't work with Node 4.x any more. I find out `loopback@3.x` has `node >= 6`.